stages/org.osbuild.grub2

```diff
@@ -137,6 +151,7 @@ def main(tree, options):
     kernel_opts = options.get("kernel_opts", "")
     legacy = options.get("legacy", True)
     uefi = options.get("uefi", None)
+    ostree = options.get("ostree", False)
```

Comment:
Is this the right name for the option? Is it really ostree specific?

Reply:
You are right, this evolved from using `ostree.{commit, osname}` to locate the efi binaries of the deployment into using the efi binaries from the build root, and now could just be `copy_efi_binaries` or something.

Reply:
I went with `uefi.install` in the end.

Reply:
One problem: the `grubenv` file gets created in the wrong place: it will be created in `<diskroot>/etc/grubenv` not in `/ostree/deploy/<stateroot>/deploy/<commit>.0/etc`. So maybe the `grub2` stage needs to have a `ostree` option after all.

Reply:
Not `/etc/grubenv` but `/etc/default/grub`, so the discussion at the selinux stage.